README.md

```diff
@@ -8,7 +8,7 @@ This is the home for the user-facing documentation and related infrastructure fo
 
 We believe in the power of collaboration and welcome contributions from the community. If you've noticed an issue or have ideas for improvement, please create a pull request or submit an issue. A member of ValidMind's documentation team will review your suggestions and support you in contributing to the docs. 
 
-- Follow the [Quickstart](https://docs.validmind.ai/guide/quickstart.html) for a 20 minute introduction to the Developer Framework and ValidMind Platform.
+- Follow the [Quickstart](https://docs.validmind.ai/guide/quickstart-try-developer-framework-with-jupyterhub.html) for a 15-minute introduction to the Developer Framework and ValidMind Platform.
```

Comment:
@nrichers This got moved, for this PR it now lives at: `site/get-started/developer/try-with-jupyterhub.qmd`

Reply:
Thanks for updating!